manager/controllers/utils/configuration.go

```diff
@@ -25,6 +25,7 @@ const (
 	PrettyLoggingKey                  string = "PRETTY_LOGGING"
 	CatalogProviderNameKey            string = "CATALOG_PROVIDER_NAME"
 	DatapathLimitKey                  string = "DATAPATH_LIMIT"
+	envVarLiteral                     string = "Env variable "
```

Comment:
Not a constant, you can remove it from the message.

Reply:
Do you mean to add lint-ignore? Or just change it to vars?